buildkite/test-template-ci.j2

```diff
@@ -411,6 +413,53 @@ steps:
         - exit_status: -10  # Agent was lost
           limit: 2
 
+  {% if branch == "main" %}
+  - label: ":docker: build image HPU"
+    key: image-build-hpu
+    depends_on: ~
+    agents:
+      queue: cpu_queue_postmerge_us_east_1
+    commands:
+      - "aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/q9t5s3a7"
+      - |
+        #!/bin/bash
+        # Fetch the compatible vLLM commit for vllm-gaudi
+        VLLM_STABLE_COMMIT=$(curl -s https://raw.githubusercontent.com/vllm-project/vllm-gaudi/main/last-good-commit-for-vllm-gaudi/VLLM_STABLE_COMMIT | tr -d '\n')
```

Comment:
can we use
```
git clone https://github.com/vllm-project/vllm-gaudi
cd vllm-gaudi
export VLLM_COMMIT_HASH=$(git show "origin/vllm/last-good-commit-for-vllm-gaudi:VLLM_STABLE_COMMIT" 2>/dev/null)
cd ..
```

I tried with the link, it looks like it is not working

<img width="978" height="166" alt="image" src="https://github.com/user-attachments/assets/e57669a2-9cea-4117-af8d-bef62e3a9bf1" />